apiary.apib

```diff
@@ -300,15 +300,121 @@ After the password is changed, the Client with be unauthorized.
                 "password_confirmation": "new_password_different"
             }
 
-+ Response 421  (application/json)
++ Response 403 (application/json)
 
         {   error: "Password and password confirmation don\'t match"    }
 
 
-+ Response 500  (application/json)
++ Response 500 (application/json)
 
         {   error: "Error updating password"    }
 
+### Delete a Credential [DELETE]
+Deletes a new credential. The current client must be an account administrator and own the credential with the given id.
+
++ Request (application/json)
+
+    + Header
+
+            Accept: application/json
+            Authorization: eyJ0eXAiOiJKV1QiLCJhbiAoOiJIUzI1NiJ9.eyJvX2F1dGhfYXBwbGljYXRpb25faWQiOjF9.h7ZNPpbd_Bu8yL27fmCAI_wgyZJI44b5eRRxwWUcwaQ
+
++ Response 200 (application/json)
+
+        {}
+
++ Response 403 (application/json)
+
+        {   errors: ["Action forbidden"]    }
+
++ Request (application/json)
+
+    + Header
+
+            Accept: application/json
+            Authorization: un_authorized_token
+
++ Response 401 (application/json)
+
+        {   error: "Not authorized"   }
+
+
+
+### Create a Credential [POST /clients/credentials]
+Creates a new credential. The current client must be an account administrator.
+
++ Request (application/json)
+
+    + Header
+
+            Accept: application/json
+            Authorization: eyJ0eXAiOiJKV1QiLCJhbiAoOiJIUzI1NiJ9.eyJvX2F1dGhfYXBwbGljYXRpb25faWQiOjF9.h7ZNPpbd_Bu8yL27fmCAI_wgyZJI44b5eRRxwWUcwaQ
+
+    + Body
+
+            {
+                "login": "12345667",
+                "password": "new_password",
+                "provider": "visa"
```

Comment:
i know it's not the case now but it comes to my attention what will the flow be for electronic credentials. Maybe a clarifying note that for electronic credentials they can ask support for help?